### ISSUE TITLE
I want to deploy model on iOS. model.model[-1].export = False throws an error for coreml conversion.

### ISSUE THREAD

**Issue by waheed0332** [ultralytics/yolov5#1106]
Can someone provide me the pipeline for IOS? I really want to deploy this on iOS.

**Comment by waheed0332** [comment]
@glenn-jocher can you shed some light on this?

**Comment by porterjenkins** [comment]
Hi @glenn-jocher I am also trying to understand this flag. It seems the `export.py` script expects the model to be in training model (`model.train()`). This has the result that the output of the forward function is less intelligible than when in eval mode (`model.eval()`). Can you explain this? How can I use the predictions from the model with CoreML?
